docs/reference/sql/alter.md

```diff
@@ -77,6 +79,31 @@ Currently following options are supported:
 ALTER TABLE monitor SET 'ttl'='1d';
 ```
 
+### Modify column fulltext index options
+
+Enable fulltext index on a column:
+
+```sql
+ALTER TABLE monitor MODIFY COLUMN load_15 SET FULLTEXT WITH (analyzer = 'Chinese', case_sensitive = 'false');
+```
+
+You can specify the following options using `FULLTEXT WITH` when enabling fulltext options:
+
+- `analyzer`: Sets the language analyzer for the full-text index. Supported values are `English` and `Chinese`. Default is `English`.
+- `case_sensitive`: Determines whether the full-text index is case-sensitive. Supported values are `true` and `false`. Default is `false`.
+
+If `WITH <options>` is not specified, `FULLTEXT` will use the default values.
+
+Disable fulltext index on a column:
+
+```sql
+ALTER TABLE monitor MODIFY COLUMN load_15 UNSET FULLTEXT;
+```
+
+The column must be a string type to alter the fulltext index.
+
+When the fulltext index is disabled, you can enable it and specify the `analyzer` and `case_sensitive` options. When the fulltext index is already enabled on a column, you can disable it but cannot modify the options.
```

Comment:
What will happen if users disable the full-text index and enable it with different options? @zhongzc

Reply:
For `UNSET` part, the previously constructed index would still take effect. For `SET` part, I guess the engine will start to construct the index and apply it once finished. cc @zhongzc for validation.

Reply:
Oh, in principle, it should not be allowed to alter options other than `enable`.

After modification, new data will use different options from old data to build indexes, which may produce some unexpected query results.

If the current implementation allows altering options other than "enable", this can be pointed out in the documentation.